lib/Backend/FlowGraph.cpp

```diff
@@ -451,18 +459,21 @@ FlowGraph::Build(void)
     }
 #endif
 
-    if (this->finallyLabelStack)
+    if (createNonExceptionFinally || this->func->IsLoopBodyInTryFinally())
```

Comment:
Under what circumstances is this condition different from the condition where finallyLabelStack is created?

Reply:
Its not, I'll change it back